Examples/Image/Classification/AlexNet/readlabels.py

```diff
@@ -0,0 +1,85 @@
+import sys
+import time
+from Read_labelclsloc import *
```

Comment:
`from foo import *` is not recommended as a practice. Maybe you can just import the module or selectively import few symbols.